code/game/machinery/bioprinter.dm

```diff
@@ -58,7 +58,7 @@
 	prints_prosthetics = 1
 
 /obj/machinery/bioprinter/attack_hand(mob/user)
-	if(!ishuman(user) || !issilicon(user))
+	if(!ishuman(user) && !issilicon(user))
```

Comment:
Any chance you can change the !ishuman to an !ishigherbeing check instead?

Reply:
sure